apps/notification/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class Notification(models.Model):
+    class NotificationType(models.IntegerChoices):
+        FIGURE_APPROVED = 0, 'Figure approved'
+        FIGURE_RE_REQUESTED_REVIEW = 1, 'Figure re-requested review'
+        FIGURE_UN_APPROVED = 2, 'Figure un-approved'
+        EVENT_ASSIGNED = 3, 'Event assigned'
+        EVENT_ASSIGNEE_CLEARED = 4, 'Event assignee cleared'
+        EVENT_SIGNED_OFF = 5, 'Event signed off'
```

Comment:
Let's use similar numbering for enums.
```suggestion
        FIGURE_APPROVED = 0, 'Figure approved'
        FIGURE_RE_REQUESTED_REVIEW = 1, 'Figure re-requested review'
        FIGURE_UN_APPROVED = 2, 'Figure un-approved'
        EVENT_ASSIGNED = 101, 'Event assigned'
        EVENT_ASSIGNEE_CLEARED = 102, 'Event assignee cleared'
        EVENT_SIGNED_OFF = 103, 'Event signed off'
```

Reply:
@tnagorra Changed